### PR TITLE
docs: misc updates to migration docs

### PR DESCRIPTION
**Pull request by AndrewKushnir**
This PR includes a few commits to update migration docs:

- Make the docs more consistent
- Removes no longer present migrations
- Adds doc for a migration that converts eager to lazy routes

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/content/reference/migrations/overview.md [hunk 1]
```diff
@@ -5,23 +5,14 @@ Learn about how you can migrate your existing angular project to the latest feat
...
   <docs-card title="Control Flow Syntax" link="Migrate now" href="reference/migrations/control-flow">
-    Control Flow Syntax is available with Angular 17 release and allows you to use more ergonomic syntax which is close to javascript, better type checking and lazy load part of the component. It replaces the need to imports CommonModule to use functionalities like `*ngFor`, `*ngIf`.
+    Built-in Control Flow Syntax allows you to use more ergonomic syntax which is close to JavaScript and has better type checking. It replaces the need to import `CommonModule` to use functionality like `*ngFor`, `*ngIf` and `*ngSwitch`.
   </docs-card>
   <docs-card title="inject() Function" link="Migrate now" href="reference/migrations/inject-function">
     Angular's `inject` function offers more accurate types and better compatibility with standard decorators, compared to constructor-based injection.
   </docs-card>
+  <docs-card title="Lazy routes" link="Migrate now" href="reference/migrations/route-lazy-loading">
+    Convert eagerly loaded component routes to lazy loaded ones. This allows to split production bundles into smaller chunks, to load less JavaScript at initial page load.
```
Comment (michael-small) [inline adev/src/content/reference/migrations/overview.md:16]:
```suggestion
    Convert eagerly loaded component routes to lazy loaded ones. This allows you to split production bundles into smaller chunks, to load less JavaScript at initial page load.
```

Comment (michael-small) [reply adev/src/content/reference/migrations/overview.md:16]: I see that the `route-lazy-loading.md` also skips the "you" too.

Comment (AndrewKushnir) [reply adev/src/content/reference/migrations/overview.md]: Thanks for the comment, I've updated the content.

Comment (michael-small) [reply adev/src/content/reference/migrations/overview.md]: Thank you

**Review by thePunderWoman — APPROVED** [review]
LGTM

**Review by michael-small — APPROVED** [review]

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit dc075a2fc6808af76a6db6d7df0a11ef230800d3.

The changes were merged into the following branches: main, 18.2.x

**Pull request closed without merging**